app/views/tutorial/a9.html

```diff
@@ -22,28 +22,44 @@ <h3 class="panel-title">Description</h3>
+                            <a href="https://docs.github.com/en/github/managing-security-vulnerabilities/configuring-dependabot-security-updates">Dependabot security updates</a> can automatically make GitHub pull requests to update vulnerable dependencies
                         </li>
                         <li>
-                            <a href="https://www.npmjs.com/package/npm-check">npm-check</a> Check for outdated, incorrect, and unused dependencies.
+                            <a href="https://snyk.io/">Snyk.io</a> is a Node.js CLI tool and Platform to scan and detect vulnerable packages
                         </li>
+                    </ol>
+                </p>
+                <p>The tools above make use of vulnerability lists, which can also be viewed directly or searched here:
+                    <ol>
                         <li>
-                            <a href="https://david-dm.org/">David DM</a> gets you an overview of your project dependencies, the version you use and the latest available, so you can quickly see what's drifting
+                            <a href="https://www.npmjs.com/advisories">NPM Security Advisories</a>
                         </li>
                         <li>
-                            <a href="https://www.npmjs.com/package/requiresafe">requireSafe</a> helps you keep your node applications secure
+                            <a href="https://github.com/advisories">GitHub Advisory Database</a>
                         </li>
                         <li>
-                            <a href="http://www.bithound.io">bithound.io</a> is a Node.js code analysis service.
+                            <a href="https://snyk.io/vuln">Snyk Vulnerability DB</a>
                         </li>
+                    </ol>
+                </p>
+                <p>There are some other tools that can detect and update outdated packages:
+                    <ol>
                         <li>
-                            <a href="#npm outdated">npm outdated</a> as well as 'yarn outdated' are both command line ways to show possibly out of date dependencies
+                            <a href="https://docs.npmjs.com/cli/v6/commands/npm-outdated">npm outdated</a> and <a href="https://classic.yarnpkg.com/en/docs/cli/outdated">yarn outdated</a> are both command line ways to show possibly out of date dependencies
+                        </li>
+                        <li>
+                            <a href="https://docs.github.com/en/code-security/supply-chain-security/about-dependabot-version-updates">Dependabot version updates</a> can automatically make GitHub pull requests to update outdated dependencies
+                        </li>
+                        <li>
+                            <a href="https://david-dm.org/">David DM</a> gets you an overview of your project dependencies, the version you use and the latest available, so you can quickly see what's drifting
```

Comment:
david-dm is an unmaintained project and we can remove it

Reply:
Is it unmaintained? I didn't see anything to suggest that at either https://david-dm.org or https://github.com/alanshaw/david-www.

Reply:
Ahh, maybe this went back into active development. I remember at point in time seeing how this was largely unmaintained.

Reply:
Seems like it has resumed activity
![image](https://user-images.githubusercontent.com/316371/113675748-5b08ec80-96c4-11eb-8268-2adbe81bf953.png)